api_docs/changelog.md

```diff
@@ -20,6 +20,24 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 10.0
 
+**Feature level 344**
+
+* [`DELETE /users/me/subscriptions`](/api/get-subscriptions): Channel
+  administrators can now unsubscribe other users even if they are not
+  an organization administrator or part of
+  `can_remove_subscribers_group`. Realm administrators are now
+  allowed to unsubscribe other users from a private channel even if
+  they are not subscribed to that channel.
+* [`PATCH /streams/{stream_id}`](/api/update-stream),
+  [`DELETE /streams/{stream_id}`](/api/archive-stream): Channel and
+  organization administrators can modify all the settings of a private
+  channel without having access to it. They cannot add subscribers to
+  the channel without having access to it.
+* [`GET /events`](/api/get-events): If a user is a channel
```

Comment:
Not sure if this applies to more endpoints

Reply:
Some other endpoints affected here are `POST /register`, `GET /users/me/subscriptions`, `GET /streams/{stream_id}/members`.